html/config/package.md

```diff
@@ -0,0 +1,14 @@
+macOS
+Stable Nightly
+brew Pkg.jil source
+C++ Python Julia All
+
+Linux (Debian)
+Stable Nightly
```

Comment:
I noticed that there are only directions for `source` for `nightly` in Linux, but when I click `nightly` on the page, it doesn't remove the other package/language options.  Should it?

Reply:
No, I have to overhaul the package matrix.